06-webpack/classworks/Malkov_Aleksey/webpack.config.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+
+
+module.exports = {
+  mode: 'production',
+  entry: {
+    main: './src/index.js'
+  },
+  output: {
+    filename: '[name].[hash].js',
+    path: path.resolve(__dirname, 'dist')
+  },
+  devServer: {
+    port: 9000
+  },
+  devtool: 'none',
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/index.html'
+    }),
+    new CleanWebpackPlugin(),
+    new MiniCssExtractPlugin(),
+    new webpack.DefinePlugin({
+      STUDENT: JSON.stringify('Aleksei Malkov')
+    }),
+    new BundleAnalyzerPlugin()
```

Comment:
If you use optimization and production mode, you don't need plugins for development (remove bundle analyzer plugin) or check how to work with process.env.NODE_ENV

Reply:
ok